tests/rifCollectorAppTest.php

```diff
@@ -0,0 +1,45 @@
+declare(strict_types=1);
+
+<?php
+
+require_once 'rif-collector-app.php';
+
+use PHPUnit\Framework\TestCase;
+
+class rifCollectorAppTest extends TestCase {
+
+    public function testPopulateTable(): void
```

Comment:
```suggestion
    public function testPopulateTable_Success(): void
```

What type of test is this? it would be conventional to use the name suggested

Reply:
Suggestion actioned, and added to notes, thank you.

Reply:
Did change to keep with current naming convention format.